server/service/src/sync/translations/requisition_line.rs

```diff
@@ -73,6 +74,9 @@ pub struct LegacyRequisitionLineRow {
 
     #[serde(rename = "Cust_loss_adjust")]
     pub stock_adjustment_in_units: f64,
+
+    #[serde(default, deserialize_with = "object_fields_as_option")]
+    pub oms_fields: Option<serde_json::Value>,
```

Comment:
I just used simple serde value type for oms_field here as of unsure about what other field(s) can be used inside. But let me know if this needs to specific type.

Reply:
@jmbrunskill @andreievg I just expanded `LegacyRequisitionLineRow` to have `expiry_date` in `oms_fields` for the OG msupply requisition line since to me it's a more like pseudo field that doesn't require to be created or directly to be used for requisition_line in the oms but just going to be used for display purposes in the OG mSupply?

But let me know if you still think of adding this as a separate field in the requisition_line or could be used differently than above.

Reply:
@andreievg 
> have you done the changes in mSupply ? We need to increment sync version for this

Have bumped sync v5 api max version - 078228206f and [79fcfe08a5](https://github.com/msupply-foundation/msupply/pull/16900/commits/79fcfe08a5421125e833c238f26d31cbdeb10a93)

> Also can you please make it a concrete type, rather the serde_json::Value, as per other oms_fields (this keeps it consistent and easier to extend)

Updated 9eeecb003e